src/features/pm/pages/PmReportsPage.tsx

```
@@ -0,0 +1,212 @@
+import { Button } from '@/components/ui/button'
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
+import { useToast } from '@/hooks/use-toast'
+
+const formatINR = (value: number) =>
+  new Intl.NumberFormat('en-IN', { style: 'currency', currency: 'INR', maximumFractionDigits: 0 }).format(value)
+
+const downloadCsv = (filename: string, rows: Record<string, unknown>[]) => {
+  const headers = Array.from(new Set(rows.flatMap((r) => Object.keys(r))))
+  const escape = (v: unknown) => {
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)